### DISCUSSION TITLE
Interactive AA installer

### DISCUSSION THREAD

**Discussion by seanlinsley** [activeadmin/activeadmin#7872] [Ideas]
Since Rails generators have the ability to pause and prompt the user for input, we should be using it to smooth over common configuration issues that people run into. This ticket will be used to keep track of and discuss new features. Here's the list so far:

- [ ] choosing the class name for the user & whether or not to use an existing class
- [ ] creating an install without authentication
- [ ] display warning if `require_tree` detected in application.css or application.js
- [x] ~~default user credentials configuration -> #3457~~ replaced by #3988

**Comment by javierjulio** [comment]
Some good ideas here. I'll leave this open if someone wants to tackle it. I think the most we should do for the `require_tree` issue is just display a warning if we detect it in either file. Otherwise its working as intended since its meant to include everything. Its just not practical to ever use require_tree since order is very important in both types. I'm afraid we are overthinking this and a simple warning is sufficient.
